pom.xml

```diff
@@ -162,9 +162,9 @@ THE SOFTWARE.
       <version>1.2</version>
     </dependency>
     <dependency>
-      <groupId>com.google.code.findbugs</groupId>
-      <artifactId>annotations</artifactId>
-      <version>3.0.1</version>
+      <groupId>com.github.spotbugs</groupId>
+      <artifactId>spotbugs-annotations</artifactId>
+      <version>3.1.12</version>
```

Comment:
Version could be added to `dependencyManagement` in Parent POM

Reply:
> Version could be added to `dependencyManagement` in Parent POM

Agreed. That seems like a pre-existing issue, though, and not something related to the switch to SpotBugs.

Reply:
> Version could be added to `dependencyManagement` in Parent POM

The version _already is_ present in a `dependencyManagement` section in the parent POM; see [here](https://github.com/jenkinsci/pom/commit/f15580982fe7cc7f486483bee01108d909940292#diff-600376dffeb79835ede4a0b285078036R110). So I removed the version from this POM, and I also changed `<scope>provided</scope>` to `<optional>true</optional>` as described in [the SpotBugs documentation](https://spotbugs.readthedocs.io/en/stable/migration.html).